Tasks/AzureResourceGroupDeploymentV2/models/DeployAzureRG.ts

```diff
@@ -99,13 +99,21 @@ export class AzureRGTaskParameters {
     
     private _getAzureADGraphCredentials(connectedService: string): msRestAzure.ApplicationTokenCredentials {
```

Comment:
Considering that this function is being called form multiple tasks, we should move it to a common location. How about moving it to azure-graph module as a static function?